spell_checker/cspell_ignored_words.txt

```diff
@@ -35,3 +35,14 @@ datev
 paytm
 mpesa
 gocardless
+SINV
+RFID
+Topup
+topup
+updation
+Unpledge
+Unsubmitted
+IFSC
+MICR
+companie
+ESOPS
```

Comment:
```suggestion
SINV
RFID
Unpledge
Unsubmitted
IFSC
MICR
ESOPS
```

Wrong words shouldn't be included here. For example, "updation" is not correct english.